caesar_cipher/run-cli.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'lib/cipher'
+
+user_message = ARGV[0]
+user_input_number = ARGV[1].to_i
+
+begin
+  Cipher.new(user_input_number).encrypt(user_message)
```

Comment:
@zzokki81 you should store the result in a variable and use it later to display it without doing the object allocation and encryption again.

Also using `rescue`/`else` is not that common. I assume you didn't use a local variable because it wouldn't show up in `else` block. Am I right?

The pattern you are looking here is:

```
begin
  # do whatever you want including printing the result
rescue
  # handle error
end
```